### DISCUSSION TITLE
Serves images with low resolution... what?

### DISCUSSION THREAD

**Discussion by Elindo586** [GoogleChrome/lighthouse#14357] [Q&A]
The actual real size of the image is 1200x630   ( it is the same picture from Open Graph) 

Lighthouse is tell me:

Display size 400 x 210  (that is what I want, and same ratio as original picture)
Actual size 400 x 210  
Expected size 600 x 315    <---- why is that????  that is just too large 

I am using Next.js for the site...   my code  for the image is this:

```
</div>
                  <Image
                    src={item.picture}
                    alt="Blog picture"
                    width={400}
                    height={210}
                  ></Image>
                  <div>


```

![lighthouse](https://user-images.githubusercontent.com/93617433/188684154-846c216e-dd3b-47ae-a484-f179d6e80393.PNG)

BTW....

Yes, I am also getting this other error.... which I believe I shouldn't be getting??? 

![lighthouse1](https://user-images.githubusercontent.com/93617433/188684460-9570ae96-60be-4ee4-8291-b61005c587f1.PNG)

**Comment by adamraine** [comment]
> Expected size 600 x 315 <---- why is that???? that is just too large

It's because Lighthouse emulates a Moto G4 which has a higher [pixel density](https://web.dev/codelab-density-descriptors/). To maximize clarity, Images should be served at higher resolution than the image element's CSS dimensions.

> Yes, I am also getting this other error.... which I believe I shouldn't be getting???

Hmm this audit should ignore `data:` URI SVGs. Are you able to share a test url for this report?

**Comment by Elindo586** [comment]
So, how can I properly fix this error?

for the original post I just made the picture bigger because I had room to make it bigger.... the picture had an actual bigger size...

Now, I am getting the same error on another picture which I wasn't getting the error.... somehow it is now showing up...

The code I had to fix it is this:
```
<div className="d-flex justify-content-center">
              <Image
                src="/images/rexrothpump-1.png"
                alt="piston-pump"
                width={263}
                height={191}
                layout="fixed"
              />
            </div>
```

^^^  I  add it `layout = fixed`  but now the picture gets distorted on mobile devices. 

The actual size of this other picture is 263x191  but the error I was getting was expecting it to be much larger

**Comment by Elindo586** [comment]
Why is google telling me I need a picture bigger than its actual size?   the picture itself is only 263x191

If I make this bigger it looks very grainy.... for this picture I can't make it bigger like I did with the other ones.

![image11](https://user-images.githubusercontent.com/93617433/188904855-6382e30d-12e0-4888-8309-0ec60f52a287.PNG)

**Reply by adamraine** [reply]
The audit is telling you to serve a higher resolution image for devices with a higher pixel density. It's not telling you to increase the size of the image element.

**Reply by Elindo586** [reply]
Oh boy...

How can I increase pixel density?  Maybe that is something I should Google? lol...

Now... how much pixel density is needed?....  the snip above seems like it is telling me to increase size as it is talking about expected size 395x287...  so to me it looks like they want a bigger picture.....  on the previous (first post) I increase the size of the picture (because it was still smaller than the original picture) and it fixed the error...

**Reply by Elindo586** [reply]
If I increase the size of the picture it fixes the error...    but I have a terrible grainy picture and it is out of dimension with the picture below of it  coming up...

I know the error says "resolution", but all the fixes from lighthouse talk about size... like using sharp and serving different sizes for different devices......   but I am also using bootstrap and the pictures do go smaller depending on the device, and they get up to whatever size its set on bigger screens...

Maybe I'll try to use a different image, but still I would want to display it in the size I have....  for some reason the next picture below of it is not getting the error...

Actually... this small picture wasn't getting an error before, but it came up just today.

**Comment by Elindo586** [comment]
BTW... the url is  www.tu.biz and https://www.tu.biz/blog/castellano

I had fixed and make the pictures of the blog bigger... but the smaller picture needs to stay small...


I also understand that lighthouse shouldn't be throwing an error on explicit height and width? ???

**Reply by adamraine** [reply]
Right this was fixed in v9.6.6, but this hasn't made it into DevTools on Chrome stable yet.
